src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -328,13 +322,8 @@ def test_FV3Forecast_run_dry_run(caplog, fv3_run_assets):
 def test_run_submit(fv3_run_assets):
     batch_script, config_file, config = fv3_run_assets
     with patch.object(FV3Forecast, "_validate", return_value=True):
-        with patch.object(forecast.subprocess, "run") as sprun:
+        with patch.object(forecast, "run") as run:
```

Comment:
The `"run"` being patched is now `run()` from `uwtools.utils.run`, which is `import`ed into the `uwtools.drivers.forecast` module.

Reply:
This is getting a bit convoluted. From the context of the `assert` below, it makes sense that the run is from `uwtools.utils.run`, but from the line `with patch.object(forecast, "run") as run:`, it's quite unclear that this isn't the `Forecast.run` method unless you read pretty hard into it.